aspnetcore/fundamentals/app-state.md

```diff
@@ -16,28 +16,26 @@ ms.custom: H1Hack27Feb2017
 
 # Introduction to session and application state in ASP.NET Core
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Steve Smith](http://ardalis.com)
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [Steve Smith](http://ardalis.com)  and [Diana LaRose](https://github.com/DianaLaRose)
```

Comment:
Hi - I didn't make that particular change, but thanks for letting me know about double spaces. I had been ignoring them because they don't render. As for serial commas: yes, they're definitely the rule.